controller/uninstall_controller.go

```diff
@@ -368,6 +387,13 @@ func (c *UninstallController) deleteCRDs() (bool, error) {
 		return true, c.deleteNodes(nodes)
 	}
 
+	if settings, err := c.ds.ListSettings(); err != nil {
```

Comment:
The same, no need for settings removal handling.

Reply:
removed